avocado/core/settings.py

```diff
@@ -144,6 +145,9 @@ def name_or_tags(self):
 
     @property
     def argparse_type(self):
+        if self._argparse_type is not None:
+            return self._argparse_type
+
```

Comment:
Any reason for setting this as private property since you are creating a setter?

In this case, if `_argparse_type` is None, it will return `None` anyway.

Reply:
My point is that this property here is doing not much, since if `self._argparse_type is None` it will return `None` as well.

Reply:
I get that... but then, if it's **not** `None`, it does something else (the previous behavior).  Maybe if you write a different proposal as code I would understand your point.